internal/docs/release.md

```diff
@@ -0,0 +1,31 @@
+# Releases
+
+We aim to do a release after each Sprint (roughly, every 3 weeks).
+
+To do a release:
+
+-   Identify the [commit](https://github.com/google/go-cloud/commits/master) you
+    want to label as the release; it should be a commit from around the end of
+    the last Sprint.
+-   Pick the new release tag; it's probably `v0.x.0` where `x` is whatever the
+    last release was plus one, but follow
+    [semantic versioning](https://semver.org/).
+-   Compile the release notes for this release by reviewing the commits since
+    the last release and listing highlights, in the form: `**<component>**:
+    <description of change, past tense>`. For example, `**blob**: Added feature
+    foo.`.
+-   Tag the commit:
+
+```bash
+$ TAG=v0.n.0
+$ COMMIT=aaaaaa
+$ git tag -a ${TAG} ${COMMIT}
+# Enter your release notes in your editor.
+$ git push upstream ${TAG}
+```
+
+-   Go to [Releases](https://github.com/google/go-cloud/releases). Click `Draft
+    a new release` and select your new tag. Enter the release notes again.
```

Comment:
Again?

Reply:
That's what I did. I'm not sure what the UI does if you Draft the release all in one step; I see notes on both the previous tags and the previous releases.

Maybe we can just leave the notes for the Tag empty, and just have them on the release?

Reply:
Changed to use "-m ${TAG}" for `git tag`.